trio/_core/_tests/test_run.py

```diff
@@ -1940,25 +1999,23 @@ async def inner():
     _core.run(inner)
 
 
-def test_Nursery_init():
+def test_Nursery_init() -> None:
     with pytest.raises(TypeError):
-        _core._run.Nursery(None, None)
+        _core._run.Nursery(None, None, False)  # type: ignore[arg-type]
```

Comment:
Why the type ignore?

Reply:
We're testing that the nursery raises if you try to construct it (I think?), but `None` isn't a valid parameter. Maybe I should tweak this to pass in valid objects instead.

Reply:
Yeah I think you should.